src/styles/components/_stars.scss

```diff
@@ -0,0 +1,29 @@
+.stars {
+  display: flex;
+  margin: 0;
+  padding: 0;
```

Comment:
If you want to reset default browser styles it's better to do it for tags and in reset.scss/_normillize.scss file.

Reply:
Pay attention to comments